### ISSUE TITLE
Were source-maps always opt-out by default?

### ISSUE THREAD

**Issue by MangelMaxime** [fable-compiler/Fable#3458]
### Description

I feel like in the past source-maps where generated by default when using Fable in Watch mode.

Is it me that have a false memory or something changed in recent version of Fable?

Should source-maps be opt-in by default when using watch mode?

**Comment by alfonsogarciacaro** [comment]
I don't remember with Fable 2 but with Fable 3 at the beginning there weren't even source maps so so I think they've been always opt-in. But I made some improvements to source maps in the latest releases and I'm ok with enabling them by default in debug mode.

**Comment by MangelMaxime** [comment]
To give a bit of context, I asked this question because some people where asking how to debug F# code in the browser with Vite.

And I found that the only things which was required is to add `--sourceMaps` to Fable invocation. I think having the sourceMaps on by default would help such people because nothing debugging would be supported out of the box by most of the bundlers.

**Comment by alfonsogarciacaro** [comment]
Makes sense! I will set the source map option by default in debug/watch mode for the next release 👍
